src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_Commit.cs

```diff
@@ -35,6 +48,16 @@ private void Commit(CompletionItem item, TextChange textChange, Model model, cha
             // into us.  However, for now, we just hope that no such craziness will occur.
             this.StopModelComputation();
 
+            var waitResult = _waitIndicator.Wait(
+                "IntelliSense",
```

Comment:
Having a wait indicator on non-explicit user invoked actions is a very poor experience.  IT also means that you might type something like "foo<tab><space>" and hte space could end up cancelling the wait dialog.  It was an explicit decision on our part that completion not be cancellable because hte cancellation experience isn't appropriate for it.